.changeset/soft-planes-appear.md

```diff
@@ -0,0 +1,5 @@
+---
+"@wso2is/console": patch
```

Comment:
bump `myaccount` and `i18N`

Reply:
added with e2ed10aaa3c1bc125cee4996a20e8ae21906bd9c